src/Files.Backend/Services/Settings/IFoldersSettingsService.cs

```diff
@@ -53,6 +78,31 @@ public interface IFoldersSettingsService : IBaseSettingsService, INotifyProperty
 		/// </summary>
 		FolderLayoutModes DefaultLayoutMode { get; set; }
 
+		/// <summary>
+		/// Gets or sets a value indicating git status column's default width
+		/// </summary>
+		double GitStatusColumnWidth { get; set; }
+
+		/// <summary>
+		/// Gets or sets a value indicating git last commit date column's default width
+		/// </summary>
+		double GitLastCommitDateColumnWidth { get; set; }
```

Comment:
```suggestion
		double GitLastCommitColumnWidth { get; set; }
```

Reply:
Please do not remove "Last" and "Date" from anywhere. If you remove them, I will suggest again in upcoming cleanup PR.

Reply:
The UI is "Last commit", I got the idea from File Explorer but it's also shorter. I'm sorry for pushing the commit without telling you first.